core/queryalgebra/model/src/main/java/org/eclipse/rdf4j/query/algebra/AggregateFunctionCall.java

```diff
@@ -30,11 +31,11 @@ public AggregateFunctionCall(ValueExpr arg, String iri, boolean distinct) {
 		this.iri = iri;
 	}
 
-	public String getURI() {
+	public String getIRI() {
 		return iri;
 	}
 
-	public void setURI(String uri) {
+	public void setIRI(String uri) {
```

Comment:
I know that the other code uses URI, but it's a long time since we moved to IRI. I also noticed that I should change the argument name too.